### ISSUE TITLE
[Issue] Some Icons don't appear when they're supposed to

### ISSUE THREAD

**Issue by superpowers04** [JannisX11/blockbench#1371]
# Issue summary
Some icons like the move icon, linear keyframe icon, and blockbench website icon refuse to appear, it looks like they're missing some classes.
## Example of broken icons with inspect element
![image](https://user-images.githubusercontent.com/38338700/161857723-d66d9cfa-faae-4ad6-ad69-7d23bf14ff8d.png) ![image](https://user-images.githubusercontent.com/38338700/161858112-b65d6e6a-7db7-419b-8882-78d5a2e92105.png) ![image](https://user-images.githubusercontent.com/38338700/161858406-1434a9d0-8375-448b-b383-260b6485a7bf.png)

## Example of working icons with inspect element
![image](https://user-images.githubusercontent.com/38338700/161857922-12ab4dd7-90f2-4745-a7f1-ffabd1086070.png) ![image](https://user-images.githubusercontent.com/38338700/161858178-a07bea4c-7911-450f-b563-5d32d4b26733.png) ![image](https://user-images.githubusercontent.com/38338700/161858478-55a066c0-cefb-4f5f-8afb-6c32b62791be.png)

# Extra Information
This happens on all formats
I am running the appimage version of BlockBench on Manjaro Linux 5.12.19-1
Reinstalling does not fix
This does not appear on BlockBench 3.X

**Comment by ShadowCK** [comment]
Same problem to me

**Comment by ShadowCK** [comment]
![image](https://user-images.githubusercontent.com/26626228/162715367-690f1432-f49f-4f4b-ad74-39232fe52eae.png)
![image](https://user-images.githubusercontent.com/26626228/162715417-8096538a-b531-464a-86cd-6bf900a7f157.png)

These icons don't show for me either

**Comment by ShadowCK** [comment]
![image](https://user-images.githubusercontent.com/26626228/162720588-b81653e3-0299-4ef0-bdb9-33b7fb491f29.png)
It appeared all these icons don't work but I don't know why and what the content property calls (like \e924, what does that exactly mean?)
They worked well in and before 4.0.5 but since 4.1.0 they broke for me.
@JannisX11

**Comment by antoinePaulinB7** [comment]
I did some digging, and icomoon.woff does not appear to load:
![image](https://user-images.githubusercontent.com/43730021/166123081-41a97d0a-b3cb-4df0-b01d-02e4651279e3.png)

**Comment by lengmutou** [comment]
I also have this problem. The most fatal thing is that the key frames of actions can't be displayed. Some of my friends have this problem, too. They don't display anything after 4.0.5, hoping to fix it. I also want to use the new version of the function/(ㄒoㄒ)/~ ~

**Comment by JannisX11** [comment]
Hey, is this still an issue in 4.2.5?

**Comment by superpowers04** [comment]
> Hey, is this still an issue in 4.2.5?
Yep, while it's fixed some icons like moving, the linear keyframe icon is still missing
![image](https://user-images.githubusercontent.com/38338700/174440116-970b4cfd-ab88-4105-92dd-f2c80220bffa.png)

**Comment by JannisX11** [comment]
Hey, is this still an issue in 4.3? Are the keyframe icons working now?

**Comment by superpowers04** [comment]
> Hey, is this still an issue in 4.3? Are the keyframe icons working now?

It seems every icon fixed except for the linear key frame icon

**Comment by BlackGlory** [comment]
4.7.4

![image](https://github.com/JannisX11/blockbench/assets/1365962/d28f96bc-0602-418c-ad6e-918f9d67c6b7)

**Comment by BlackGlory** [comment]
Investigated the issue, I think it's just because of the terrible font rendering in Electron or Chromium, I would suggest replacing them with SVG icons.
